config/schemas/config.yaml

```
@@ -4685,6 +4685,57 @@ properties:
             items:
               title: S3 Bucket Name
               type: string
+          buckets:
+            title: S3 Bucket Specific Alerts
+            description: Definitions for specific S3 bucket alerts.
+            items: {}
+            prefixItems:
+            - title: S3 Bucket Specific Alert
```

```suggestion
            items:
              title: S3 Bucket Specific Alert
```

otherwise only the first item is validated.

if this was from the schema generator script then it's because it doesn't handle multiple items very well so one has to merge them as a post-processing step

---

Fixed in [94502b0](https://github.com/elastisys/compliantkubernetes-apps/commit/94502b0ebdc4a359a0c869081c9d05e68867392e) :+1: